lints/lint_idn_dnsname_must_be_nfc.go

```diff
@@ -53,11 +53,11 @@ func (l *IDNNotNFKC) Execute(c *x509.Certificate) *LintResult {
 
 func init() {
 	RegisterLint(&Lint{
-		Name:          "e_international_dns_name_not_nfkc",
-		Description:   "Internationalized DNSNames must be normalized by unicode normalization form KC",
-		Citation:      "RFC 3490",
-		Source:        RFC5280,
-		EffectiveDate: util.RFC3490Date,
-		Lint:          &IDNNotNFKC{},
+		Name:          "e_international_dns_name_not_nfc",
+		Description:   "Internationalized DNSNames must be normalized by unicode normalization form C",
+		Citation:      "RFC 5890",
+		Source:        RFC5891,
+		EffectiveDate: util.RFC5890Date,
```

Comment:
I believe this should cite and use the date for RFC 8399, which is what updates 5280 and by extension the BRs.

Reply:
Can someone take a quick look as to why exactly the build is failing? I'm not seeing the same error locally.